app/src/main/java/com/client/fire_and_water/FriendsFragment.kt

```diff
@@ -0,0 +1,78 @@
+package com.client.fire_and_water
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.*
+import com.client.fire_and_water.databinding.FragmentFriendsBinding
+
+class FriendsFragment : Fragment() {
+
+    lateinit var listAdapter: ArrayAdapter<String>
+    lateinit var friendNames: ArrayList<String>
+    private var _binding: FragmentFriendsBinding? = null
```

Comment:
If the `binding` property is private, then it is not necessary to make two of them